### ISSUE TITLE
Cardify All The List Items!

### ISSUE THREAD

**Issue by phil-lopreiato** [the-blue-alliance/the-blue-alliance-android#491]
@nwalters512 thoughts on applying pretty cards to more things?

**Comment by nwalters512** [comment]
As usual, I will defer to the [spec](https://www.google.com/design/spec/components/cards.html) (#praiseduarte):
```
Cards are a convenient means of displaying content composed of different elements.
They’re also well-suited for showcasing elements whose size or supported actions vary,
like photos with captions of variable length.
.
.
.
A quickly scannable list, instead of cards, is an appropriate way to represent 
homogeneous content that doesn't have many actions.
```
Cards are beautiful <3 but I'm going to resist the urge to make everything a card because of that. Cards work well for gameday because of the heterogeneous nature of the list, but for everything else, cards would add distracting padding and shadows where none is required.

**Comment by phil-lopreiato** [comment]
The award list and info pages are carded (ish), those fragments look a ton better than the others. It doesn't have to be full blown cards, but I think some subtle shadowing, minor padding, and color will go a long way in making it all look better. I definitely agree we shouldn't go overboard though

**Comment by nwalters512** [comment]
Again, those pages work well because content isn't homogenous. I can mock up some stuff though, if needed.

**Comment by phil-lopreiato** [comment]
Yeah, that's true. I think the inconsistency between them makes the non-carded things stand out more (not in a good way), but I know my eye for design is bad enough not to trust.

I'd say try it and see if some subtle touches can make the difference less obvious.

**Comment by nwalters512** [comment]
I agree that cards/lists side by side in tabs isn't ideal... I can mess around with it and see what I come up with.
